ansible/roles/test/tasks/advanced-reboot.yml

```diff
@@ -114,7 +117,7 @@
           become: true
 
         - set_fact:
-            new_image_location: '/tmp/new_sonic_image.bin'
+            new_image_location: '/tmp/new_sonic_image.img'
```

Comment:
>img [](start = 54, length = 3)

.img is not generic either. You may add some code comment to explain the problem. It's ok to keep original code.